classes/tweet.rb

```diff
@@ -1,4 +1,47 @@
 class Tweet
-  attr_accessor :content, :location, :created_at, :zombie, :tweet_id
+  attr_accessor :content, :location, :created_at,
+                        :zombie, :unique_id, :favs, :retweets,
+                        :original_tweet, :retweeted
+
+  # Class level attributes
+  @@tweet_id = 1
```

Comment:
@@tweet_id is the counter that we used to assign a number for the unique_id. We opted for this as opposed to something like @@all_tweets.length + 1 because even if a tweet is deleted we don't want to re-assign that unique_id.